### ISSUE TITLE
behat 3.0

### ISSUE THREAD

**Issue by jesavro** [Behat/YiiExtension#12]
The composer.json requires behat 2.5. Can behat 3.0 be used? If so can the composer file be updated?

**Comment by stof** [comment]
No, the extension has not been migrated yet to Behat 3 (Behat 3 is a complete rewrite of Behat)

**Comment by acoulton** [comment]
This extension has never been updated to Behat 3, and the Behat core team does not include any Yii users with resource to upgrade/maintain it. Therefore we will be marking it as abandoned. There is an in-progress PR at #13 which may provide some pointers should anyone wish to fork and release it as a community extension.
